code/modules/admin/verbs/SDQL2/useful_procs.dm

```diff
@@ -27,3 +27,219 @@ CALL global.json_to_object_arbitrary_vars("{'type':'/obj/item/crowbar', 'color':
 		thing.vars[attribute] = data[attribute]
 
 	return thing
+
+// Wrappers for BYOND default procs which can't directly be called by call().
+
+/proc/_abs(A)
+	return abs(A)
+
+/proc/_animate(atom/A, set_vars, time = 10, loop = 1, easing = LINEAR_EASING, flags = null)
+	var/mutable_appearance/MA = new()
+	for(var/v in set_vars)
+		MA.vars[v] = set_vars[v]
+	animate(A, appearance = MA, time, loop, easing, flags)
+
+/proc/_acrccos(A)
```

Comment:
Is this a typo?

Reply:
I don't think so... It's arc cosine, so the two cs are intentional. It does look weird tho.

Reply:
That would be **arc**cos then, wouldn't it?

Reply:
no that's fine... there's a c between the a and r

Reply:
Yeah, rogue c

Reply:
Oh, wow, I read that so many times and my brain just ignored it. Good catch!